client/src/components/Profil/DetailPage.js

```diff
@@ -1,213 +1,121 @@
-              source={source}
-            />
-          </section>
-        </Col>
-        <Col tag="section">
-          <DetailCadastralTable cadastral={this.state.cadastral} onParcelShow={this.goMap} />
-        </Col>
-      </Row>
-    ) : null
+import type {State} from '../../state'
 
-    return (
-      <Container className="Profile">
-        <Row tag="header" key="title" className="header profile-header">
-          <Col>
-            <h1 className="title">
-              <NavLink to="/profil">
-                <span className="bolder">profil</span>.verejne.digital
-              </NavLink>
-            </h1>
-          </Col>
-        </Row>
-        <Cardboard key="cardboard" politician={this.state.politician} />
-        {politician}
-        <Row key="map" id="map" className="profile-map">
-          <Col>
-            <MapContainer
-              assets={this.state.cadastral}
-              center={this.state.mapCenter}
-              zoom={this.state.zoom}
-              ref="map"
-            />
-          </Col>
-        </Row>
-      </Container>
-    )
-  }
+export type ProfileDetailPageProps = {
+  match: {
+    params: {
+      year: ?string,
+    },
+  },
 }
 
-export default DetailPage
+const DetailPage = ({assetsYears, selectedYear, assets, politician, cadastral, mapCenter}) => [
+  <Row tag="header" key="title" className="header profile-header">
+    <Col>
+      <h1 className="title">
+        <NavLink to="/profil">
+          <span className="bolder">profil</span>.verejne.digital
+        </NavLink>
+      </h1>
+    </Col>
+  </Row>,
+  <Cardboard key="cardboard" politician={politician} />,
+  <Row tag="article" key="politician" className="profile">
+    <Col tag="section">
+      <div className="profile-tabs">
+        {assetsYears.map((y) => (
+          <Link
+            to={{search: `?year=${y}`}}
+            className={y === selectedYear ? 'tab active' : 'tab'}
+            key={y}
+          >
+            {y}
+          </Link>
+        ))}
+      </div>
+      <section>
+        <DetailAsset
+          assets={assets.unmovable_assets}
+          year={selectedYear}
+          title="Majetkové priznanie: Nehnuteľnosti"
+          image={`https://verejne.digital/img/majetok/${politician.surname}_${
+            politician.firstname
+          }.png`}
+          source={assets.source}
+        />
+      </section>
+      <section>
+        <DetailAsset
+          assets={assets.movable_assets}
+          year={selectedYear}
+          title="Majetkové priznanie: Hnuteľný majetok"
+          source={assets.source}
+        />
+      </section>
+      <section>
+        <DetailAsset
+          assets={assets.income_assets}
+          year={selectedYear}
+          title="Majetkové priznanie: ostatné"
+          source={assets.source}
+        />
+      </section>
+    </Col>
+    <Col tag="section">
+      <DetailCadastralTable cadastral={cadastral} onParcelShow={this.goMap} />
```

Comment:
Pagination here as well.

Reply:
I'm not 100% sure about pagination here - we have no searching on this, having all on one page gives you the option to look for something by ctrl+f - pagination can often be user unfriendly

it's not like we'd be saving network (we already have all the data loaded) or displaying 'heavy' react components that would be slowing the page down, displaying a table with tens to hundreds rows isn't a problem even on mobile devices

Reply:
I think that long scroll bar is not a good user exp. I think seqrxh funcrionality will be part of the appllication